developer-experience/recipes/cross-platform-tasks.md

```diff
@@ -0,0 +1,54 @@
+# Cross Platform Tasks with VS Code
```

Comment:
If we omit `vs code` we can also include a reference to dockerizing builds and tests. Might help out if VS code goes out of style. Will also be relevant for people that do not use VS code (i.e., java developers)

Reply:
Great feedback, thank you! As you mentioned, VS code task options is not the only way to reduce some of the cross compatibility issues and I have rearranged the sections to state that more clearly.